orion_mcp.py

```
@@ -663,6 +751,159 @@ def main():
+# REST API Endpoints (Simple HTTP JSON API)
+# These endpoints provide direct HTTP access to Orion functionality,
+# bypassing the MCP protocol for simpler integration.
+# Uses FastMCP's @custom_route decorator.
+# =============================================================================
+
+from starlette.requests import Request
+from starlette.responses import JSONResponse
+
+
+@mcp.custom_route("/api/configs", methods=["GET"])
+async def api_get_configs(request: Request) -> JSONResponse:
```

Ruff (0.14.14)

[warning] Unused function argument: `request`

(ARG001)